kratos/includes/variables.h

```diff
@@ -470,6 +470,10 @@ namespace Kratos
     //for Geometry Variables
     KRATOS_DEFINE_3D_VARIABLE_WITH_COMPONENTS(PARAMETER_2D_COORDINATES)
 
+    // Variational redistance
+    KRATOS_DEFINE_VARIABLE(double, VARIATIONAL_REDISTANCE_COEFFICIENT_1)
```

Comment:
I don't like the names....

Reply:
Ok, I will rename them, if nobody has a better option